src/Modifiers/CoreModifiers.php

```diff
@@ -1276,17 +1280,6 @@ public function isUrl($value)
         return Str::isUrl($value);
     }
 
-    /**
-     * Returns true if the string is an external URL.
-     *
-     * @param $value
-     * @return bool
-     */
-    public function isExternalUrl($value)
-    {
-        return Str::isUrl($value) && URL::isExternal($value);
-    }
-
```

Comment:
Not sure why this was removed. Maybe you need to merge 4.x into this branch?

Reply:
Weird, I had merged 4.x in but something went strange.